examples/basic/hello-world.ts

```diff
@@ -65,8 +74,8 @@ import { pinoPrettyLogger } from "@golem-sdk/pino-logger";
     await proposalPool.remove(draftProposal);
 
     // This will keep the script waiting for payments etc
+    // Also it will release the allocation internally
```

Comment:
why? Allocations are reused across leases, so if we terminate one all others will no longer be able to pay their invoices?

Reply:
You're right. That's a leftover after one of the changes. I dropped the comment and added `await glm.payment.releaseAllocation(allocation);` below `finalize` call in the example.